chassis/configuration/configuration/reporter/session.comparison.conf.xml

```diff
@@ -77,13 +61,6 @@
 
     <!-- Monitoring -->
     <!-- Deprecated. Not used starting from 1.2.5 -->
-    <bean id="monitoringFeatureComparator"
-          class="com.griddynamics.jagger.engine.e1.sessioncomparation.monitoring.MonitoringFeatureComparator">
-
-        <property name="monitoringParameterDecisionMaker" ref="stdDevMonitoringParameterDecisionMaker"/>
-        <property name="monitoringSummaryRetriever" ref="monitoringSummaryRetriever"/>
-        <property name="sessionFactory" ref="sessionFactory"/>
-    </bean>
     <bean id="stdDevMonitoringParameterDecisionMaker"
           class="com.griddynamics.jagger.engine.e1.sessioncomparation.monitoring.StdDevMonitoringParameterDecisionMaker">
 
```

Comment:
Should it be removed?
I see that sessionComparator is used in reporting.conf.xml